evansmusomi/burger-builder
# Restricts Making and Viewing Orders to Authenticated Users

*What this does*
- Adds a spinner to the auth screen when processing
- Automatically logs a user out when their token expires
- Enables logged in users to access protected resources

*How to verify*
- A user can see a spinner when their auth request is being processed
- A logged in user is logged out automatically after 1 hour (when their token expires)
- An authenticated user can view and make orders while other users cannot
